libs/design-system-components/src/components/DynamicInfiniteScroll/index.tsx

```diff
@@ -110,7 +110,7 @@ const DynamicInfiniteScroll: React.FC<DynamicInfiniteScrollType> = props => {
   const virtualListUi = (
     <Card
       ref={parentRef}
-      customStyle={`relative min-h-[${totalSize + loadingPlaceholderSize}px] ${customStyle}`}
```

Comment:
Also remember to cleanup unused vars, if any in the next PR

Reply:
Those will be used in the next PR. I need them for the spinner. For now the spinner will introduce some issues I have commented it out.